src/org/labkey/test/LabKeySiteWrapper.java

```diff
@@ -154,6 +155,13 @@ public void simpleSignIn()
             {
                 if (isElementPresent(SiteNavBar.Locators.userMenu))
                     return true;
+
+                if(isElementPresent(Locator.tagWithId("a", "user-menu-dropdown")))
+                {
+                    goToHome();
+                    return true;
+                }
+
```

Comment:
This should make sure the standard userMenu is present after navigating. Either remove the return line and move this `if` to the start of the `waitFor`; or have it assert that the standard userMenu is present after navigating home.

Reply:
Fixed.